README.md

```diff
@@ -155,6 +169,12 @@ branches:
         apps: []
         users: []
         teams: []
+      # Permits force pushes for all users with push access. Set to null to disable.
+      allow_force_pushes:
+      # Allows users with push access to delete matching branches. Set to false to disable.
+      allow_deletions: false
```

Comment:
disable doesnt really make sense for this one either

Reply:
same [question as above](https://github.com/probot/settings/pull/480#discussion_r684915299), would you like to change "disable" to "allow"?